libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -787,8 +787,10 @@ string ASTJsonConverter::functionCallKind(FunctionCallKind _kind)
 		return "typeConversion";
 	case FunctionCallKind::StructConstructorCall:
 		return "structConstructorCall";
+	case FunctionCallKind::Unset:
+		BOOST_THROW_EXCEPTION(InternalCompilerError() << errinfo_comment("FunctionCallKind was not set."));
 	default:
-		BOOST_THROW_EXCEPTION(InternalCompilerError() << errinfo_comment("Unknown kind of function call ."));
+		BOOST_THROW_EXCEPTION(InternalCompilerError() << errinfo_comment("Unknown kind of function call."));
```

Comment:
Do you want to create a separate PR for these two changes above? Could be merged right away.